.gitlab-ci.yml

```diff
@@ -71,7 +75,7 @@ stages:
     include:
       - local: '.gitlab/custom-jobs-and-variables.yml'
       - project: 'radiuss/radiuss-shared-ci'
-        ref: 'v2023.10.0'
+        ref: 'woptim/reproducer'
```

Comment:
Is this going to be updated in for the next release?

Reply:
@MrBurmark looks like I forgot to push my latest commit. Should be pointing at v2023.12.0